🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** N/A
- **Commit:** N/A
- **PR:** N/A (direct push)

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 20440087320
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.


___

### **PR Type**
Documentation


___

### **Description**
- Add Tyk Sync 2.1.5 release notes

- Update overview to latest 2.1.5

- Provide Docker links and pull example

- Mirror changes in nightly docs


___

### Diagram Walkthrough


```mermaid
flowchart LR
  overview["Release overview (stable)"] -- bump latest to 2.1.5 --> syncnotes["Sync 2.1.5 notes (stable)"]
  nightly_overview["Release overview (nightly)"] -- bump latest to 2.1.5 --> nightly_syncnotes["Sync 2.1.5 notes (nightly)"]
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>overview.mdx</strong><dd><code>Update stable overview to Tyk Sync 2.1.5</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

developer-support/release-notes/overview.mdx

<ul><li>Bump Tyk Sync latest to <code>2.1.5</code><br> <li> Update <code>latestData.name</code> and date for 2.1.5<br> <li> Update Docker tag link to <code>v2.1.5</code></ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1256/files#diff-dd436baaf553c4f68b417769481419779095437ed991a35723c1067a6af182a6">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>sync.mdx</strong><dd><code>Add detailed Tyk Sync 2.1.5 release notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

developer-support/release-notes/sync.mdx

<ul><li>Add full section for <code>2.1.5</code> release<br> <li> Note LTS compatibility and no changes<br> <li> Provide upgrade guidance and Docker pull<br> <li> Add changelog anchor and "No changes" note</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1256/files#diff-cac9d708983eaefd5fd0b06ac25b0f3934804430ff37b7fb7d12c44c4d46e799">+29/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>overview.mdx</strong><dd><code>Update nightly overview to Tyk Sync 2.1.5</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/developer-support/release-notes/overview.mdx

<ul><li>Bump Tyk Sync latest to <code>2.1.5</code><br> <li> Update <code>latestData.name</code> and date for 2.1.5<br> <li> Update Docker tag link to <code>v2.1.5</code></ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1256/files#diff-fb8ca7bde79e266244d0bcf78f942b283149eefc86d9437a7ea05f28c48c930d">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>sync.mdx</strong><dd><code>Add nightly Tyk Sync 2.1.5 release notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/developer-support/release-notes/sync.mdx

<ul><li>Add nightly <code>2.1.5</code> release section<br> <li> Link to nightly dashboard LTS notes<br> <li> Provide upgrade guidance and Docker pull<br> <li> Add changelog anchor and "No changes" note</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1256/files#diff-78eeacba33e4f8063c2bcc4086f534b9a0dc58c55340a13d635ae6eb2a35d6ac">+29/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

joshblakeley commented:
This is out for tib so merge please @marksou

sredxny commented:
I just made a slight update regarding certificate for saml, and now this is ready to go live

marksou commented:
Note that I'm going to merge this into the `new_design` branch, which will be merged into master early next week

joshblakeley commented:
Thanks Mark

buger commented:
/release to master

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
